### PR TITLE
Fix #186, #194 and #195

### LINKED ISSUE #186: [PE-D][Tester A] Clone feature may be unnecessary
No further details regarding application of clone command was given. May not be as relevant to a financial adviser.

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: AnnabelTing/ped#7

### LINKED ISSUE #194: [PE-D][Tester E] Clone error message differ from the UG
Got these errors messages for clone 0 and clone 2147483647:

Instead of the error message mentioned in the UG:

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: wnchan/ped#5

### LINKED ISSUE #195: [PE-D][Tester E] Clone feature UG screenshots does not reflect correct GUI of that command
Wrong screenshots is being used for both cases.

In the screenshot, clone 1 lists all people.

In the screenshot, clone 1 shows out of range error message.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: wnchan/ped#6

### PR DESCRIPTION
**Pull request by bhnuka**

### CODE REVIEW

**Comment by bhnuka** [comment]
Good idea! Updated the screenshot. Thanks Syam!

**Review by syamfarh — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -925,8 +925,9 @@ If your changes to the data file makes its format invalid, FAPro will discard al
...
-| **Parameter**       | values inputted by the user.<br/>e.g. NAME, OCCUPATION, ADDRESS                                                                                                                         |
-| **Prefix**          | word that is added in front of parameter.<br/>e.g. n/, o/, a/                                                                                                                           |
-
+| Word                 | Meaning                                                                                                                         |
+|----------------------|---------------------------------------------------------------------------------------------------------------------------------|
+| **Parameter**        | values inputted by the user.<br/>e.g. NAME, OCCUPATION, ADDRESS                                                                 |
+| **Positive Integer** | An integer that is positive (i.e greater than 0). Please note that we are excluding 0 as a positive integer.                    |
```
Comment (syamfarh) [inline docs/UserGuide.md:945]: fixed for issue #194
hopefully the glossary helped to clarify some things to some people

File: docs/UserGuide.md [hunk 2]
```diff
@@ -242,7 +242,7 @@ Multiple values specified for the following single-valued field(s): n/
 
 ### Cloning a person : `clone`
 
-Clones a contact from FAPro at the specified index.
+Clones a contact from FAPro at the specified index. Makes it easier for the user when adding a contact with very similar details (i.e from the same household)
```
Comment (syamfarh) [inline docs/UserGuide.md:245]: fixed for issue #186

**Review by syamfarh — COMMENTED** [review]
LGTM! Do you plan to change the screenshot for issue #195 since some people thinks `clone 1` output `listed all persons` instead? Otherwise, I can merge if no changes

**Pull request merged**